src/java.base/share/classes/java/util/concurrent/AbstractExecutorService.java

```diff
@@ -119,7 +119,7 @@ protected <T> RunnableFuture<T> newTaskFor(Callable<T> callable) {
      */
     @Override
     public Future<?> submit(Runnable task) {
-        if (task == null) throw new NullPointerException();
+        if (task == null) throw new NullPointerException("task can't be null");
```

Comment:
Why not use `Objects.requireNonNull()` and break out the repeated code into a common method?

Reply:
I'd prefer `Objects.requireNonNull(task, "task");`